### DISCUSSION TITLE
plot_predictions not working for 01_pytorch_workflow

### DISCUSSION THREAD

**Discussion by ethvor** [mrdbourke/pytorch-deep-learning#733] [Q&A]
I am using dataspell as my ide. I have been trying to get this to work for a few days now, but when I execute all, the execution stops at 
plot_predictions(); I thought it was an issue with my own code, but when I imported the 01_pytorch_workflow file from the repo, it had the same issue. I have reinstalled matplotlib. I am in python 3.11 in a conda env. I am somewhat of a python noob, so I could use any help I can get.
![Screenshot 2023-11-25 155451](https://github.com/mrdbourke/pytorch-deep-learning/assets/118936291/36fa47d2-61ab-42d1-a5c2-71aa5373e6a9)

the screenshot is the result of running all cells. it stops at that line, and it never stops or shows any errors, or shows a plot. Thanks for your help!

**Comment by bhuvanmdev** [comment]
> I am using dataspell as my ide. I have been trying to get this to work for a few days now, but when I execute all, the execution stops at plot_predictions(); I thought it was an issue with my own code, but when I imported the 01_pytorch_workflow file from the repo, it had the same issue. I have reinstalled matplotlib. I am in python 3.11 in a conda env. I am somewhat of a python noob, so I could use any help I can get. ![Screenshot 2023-11-25 155451](https://private-user-images.githubusercontent.com/118936291/285615602-36fa47d2-61ab-42d1-a5c2-71aa5373e6a9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIyMTU1MTYsIm5iZiI6MTcwMjIxNTIxNiwicGF0aCI6Ii8xMTg5MzYyOTEvMjg1NjE1NjAyLTM2ZmE0N2QyLTYxYWItNDJkMS1hNWMyLTcxYWE1MzczZTZhOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxMFQxMzMzMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MjUyOTIwMzVlM2M5NTRlM2RmYjAwOGFiZjZjMDRkOTMzMDRhOTE0MDYxYzRmMmU5M2FhZmYwZTRkY2Y4Y2UzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Ez6zAJhqs_ZbnRiM11tDviIxP9NGmQeYdbpxko8qxrU)
> 
> the screenshot is the result of running all cells. it stops at that line, and it never stops or shows any errors, or shows a plot. Thanks for your help!

try to put a line "plt.show()" after calling the function. that might fix it.

**Comment by shisirkha** [comment]
We could help if you give us the error log from when running it.

I think you have some problems with your setup for this. @ethvor
